hail/python/hail/plot/plots.py

```diff
@@ -1573,7 +1573,7 @@ def manhattan(pvals: 'Float64Expression',
     source_pd['_contig'] = [locus.split(":")[0] for locus in source_pd['locus']]
 
     observed_contigs = [
-        contig for contig in ref.contigs.copy()
+        contig.replace("chr", "") for contig in ref.contigs.copy()
         if contig in set(source_pd['_contig'])
     ]
 
```

Comment:
this next line is gonna fail b/c the contains aren't valid anymore. I think you'll have to replace this at the very latest moment: when you construct the major_label_overrides.